src/float/mod.rs

```diff
@@ -1,7 +1,8 @@
 use num_traits::float::FloatCore as NumFloatCore;
 use rustfft::FFTnum;
+use std::fmt::Display;
```

Comment:
Why do we need the `Display` trait? Is it gonna be ok when targeting wasm?

Reply:
I put it in there when debugging, because I couldn't print things without it...

Is there a way to do a WASM build to make sure it works? It would be nice if this could be part of the checks so we wouldn't need to worry about accidentally introducing WASM-incompatible things...

Reply:
Yes, you can test the `pitch-detection-app` against the local dev version of the library:

In the `Cargo.toml` in the `wasm` folder of the `pitch-detection-app` replace `pitch-detection = "0.2"` with `pitch-detection = { path = "/path/to/pitch-detection" }`

After that you'll have to re-run the `wasm-pack` command, and possibly rebuild the client as well

Reply:
Okay. I built it and everything worked :-).